tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -44,6 +44,17 @@ var _ = Describe("odo devfile push command tests", func() {
 
 	Context("Verify devfile push works", func() {
 
+		It("Check that the experimental warning appears for create and push", func() {
+			helper.CmdShouldPass("odo", "preference", "set", "Experimental", "true")
+			helper.CopyExample(filepath.Join("source", "nodejs"), context)
+
+			// Check that it will contain the experimental mode output
+			experimentalOutputMsg := "Experimental mode is enabled, use at your own risk"
+			Expect(helper.CmdShouldPass("odo", "create", "nodejs")).To(ContainSubstring(experimentalOutputMsg))
+			Expect(helper.CmdShouldPass("odo", "push")).To(ContainSubstring(experimentalOutputMsg))
+
```

Comment:
also a check that when experimental is turned off, this string is gone

Reply:
Ah yes, I'll add that.